resources/geocoding/en/54.txt

```diff
@@ -13,224 +13,311 @@
 # limitations under the License.
 
 # Translated from es/54.txt using Freebase and Wikipedia.
+# https://es.wikipedia.org/wiki/N%C3%BAmeros_telef%C3%B3nicos_en_Argentina
 # Some manual edits were made after the file was generated, according to the
 # ITU file published 2011-12-16 here: http://www.itu.int/oth/T0202000009/en
```

Comment:
Remove these 4 lines and please replace it with below lines as below,

\# Historically translated from es/54.txt using Freebase and Wikipedia.
\# Some manual edits were made after the file was generated, according to the
\# ITU file published 2011-12-16 here: http://www.itu.int/oth/T0202000009/en
\# Recently updated by consulting
\# https://es.wikipedia.org/wiki/N%C3%BAmeros_telef%C3%B3nicos_en_Argentina